compiler-rt/lib/builtins/CMakeLists.txt

```diff
@@ -892,6 +892,18 @@ else ()
         ${TARGET_${arch}_CFLAGS})
       list(JOIN CMAKE_REQUIRED_FLAGS " " CMAKE_REQUIRED_FLAGS)
       message(STATUS "Performing additional configure checks with target flags: ${CMAKE_REQUIRED_FLAGS}")
+      # For ARM archs, exclude any sync builtins if dmb or mcr p15, #0, r0, c7, c10, #5
```

Comment:
I would rephrase the comment and the error message to be more clear about the overall intent. From the comment and the error message I would think that we're testing for the DMB instruction specifically. Perhaps that was the case in an earlier iteration of the patch?

For the comment I'd write something like:
`Exclude sync builtins for Arm architectures before Armv6`

And for the error I'd write something like:
`#error sync builtins are not supported prior to Armv6`